EventFilter/EcalTBRawToDigi/src/CamacTBDataFormatter.cc

```diff
@@ -168,7 +166,7 @@ void CamacTBDataFormatter::interpretRawData( const FEDRawData & fedData,
   LogDebug("CamacTBDataFormatter") << "event number in spill: "<< b;
 
   a = buffer[wordCounter];wordCounter++;
-  b = (a& 0xffffff);
+  b = (a& 0xffff);
```

Comment:
Why did `0xffffff` get changed to `0xffff`? Such a change could zero out the high bits of `b`, which could affect run-time behavior.